tracer/contrib/redigo/tracedredis.go

```diff
@@ -0,0 +1,104 @@
+package tracedredis
+
+import (
+	"bytes"
+	"context"
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/DataDog/dd-trace-go/tracer/ext"
+	redis "github.com/garyburd/redigo/redis"
+	"net"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+type TracedConn struct {
+	redis.Conn
+	tracer  *tracer.Tracer
```

Comment:
wrap host, port, network, service, tracer in a TraceParams struct and pass it to both of these . avoids conflicts in field names